src/classes/STG_InstallScript.cls

```diff
@@ -63,6 +64,8 @@ global without sharing class STG_InstallScript implements InstallHandler {
             
             executeAction(InstallAction.RUN_NEW_ORG_SCRIPT, logger);
 
+            // Create the Opportunity Stage to State mapping record for finalized
+            executeAction(InstallAction.CREATE_FINALIZED_OPP_STAGE_TO_STATE_RECORD, logger);
         // Upgrade of existing Cumulus installation
```

Comment:
If there are no states mapped at all, it should map Committed to Pledged, if a Pledged Stage exists.
If there is no finalized states mapped and the Opp Stage for When Fully Paid has a value, it should map that

If the criteria are not met, it should not attempt to execute these.